src/erlcloud_elb.erl

```diff
@@ -231,6 +244,176 @@ next_token(Path, XML) ->
             ok
     end.
 
+%% --------------------------------------------------------------------
+%% @doc Calls describe_load_balancer_policies([], 
+%%              default_configuration())
+%% @end
+%% --------------------------------------------------------------------
+describe_load_balancer_policies() ->
+    describe_load_balancer_policies([], default_config()).
+
+%% --------------------------------------------------------------------
+%% @doc describe_load_balancer_policies with specific policy names.
+%% @end
+%% --------------------------------------------------------------------
+describe_load_balancer_policies(PolicyNames) when is_list(PolicyNames) ->
+    describe_load_balancer_policies(PolicyNames, default_config());
+describe_load_balancer_policies(Config) when is_record(Config, aws_config) ->
```

Comment:
just code-style wise: why guard instead of a match for a record?  
e.g.`describe_load_balancer_policies(Config = #aws_config{})`